CHANGES.md

```diff
@@ -6,6 +6,8 @@
 - Stop pickling the annotations of a dynamic class for Python < 3.6
   (follow up on #276)
   ([issue #347](https://github.com/cloudpipe/cloudpickle/issues/347))
+- Add support for pickling dynamic `typing.TypeVar`s
```

Comment:
Nitpick: I would re-phrase this as:

Fix a bug affecting the pickling of dynamic `TypeVar` instances on Python 3.7+, and expand the support for pickling `TypeVar` instances (dynamic or non-dynamic) to Python 3.5-6.